sklearn/datasets/base.py

```diff
@@ -255,14 +255,22 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+     ----------
+    return_X_y : boolean, default=False.
+        If True, returns (data, target) instead of a Bunch object.
+        See below for more information about the `data` and `target` object.
+
     Returns
     -------
     data : Bunch
         Dictionary-like object, the interesting attributes are:
         'data', the data to learn, 'target', the classification labels,
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
-        full description of the dataset.
+        full description of the dataset. This is test text.
```

Comment:
hey, Quick question:
Initially, I installed using ```make``` rather than using ```python setup.py build_ext --inplace```

That shouldn't make any difference. Right ?